rocketpy/simulation/monte_carlo.py

```diff
@@ -722,9 +722,12 @@ def set_processed_results(self):
             try:
                 mean = np.mean(values)
                 stdev = np.std(values)
-                self.processed_results[result] = (mean, stdev)
+                pi_low = np.quantile(values, 0.025)
+                pi_high = np.quantile(values, 0.975)
+                median = np.median(values)
+                self.processed_results[result] = (mean, median, stdev, pi_low, pi_high)
             except TypeError:
-                self.processed_results[result] = (None, None)
+                self.processed_results[result] = (None, None, None, None, None)
 
     # Import methods
```

Comment:
```python
try:
    mean = np.mean(values)
    stdev = np.std(values)
    self.processed_results[result] = (mean, stdev)
    pi_low = np.quantile(values, 0.025)
    pi_high = np.quantile(values, 0.975)
    median = np.median(values)
except TypeError:
    mean = None
    stdev = None
    pi_low = None
    pi_high = None
    median = None

 self.processed_results[result] = (mean, median, stdev, pi_low, pi_high)
```

Alternatively, we could make 5 try/except blocks instead of one single.

Reply:
Oh yeah sorry i missed that, I have committed this suggestion

I don't think 5 try/except blocks is necessary. A situation where only one of these calculations fails seems unlikely to me. ie: either they all fail or none fail so catching them all in the same makes sense.